src/modules/Layout/index.tsx

```diff
@@ -8,14 +8,12 @@ type Props = {
   location: string;
 };
 
-export class Layout extends React.Component<Props, unknown> {
-  render() {
-    const { children, componentName, location } = this.props;
-    return (
-      <S.LayoutWrapper>
-        <Header location={location} componentName={componentName} />
-        <main role="main">{children}</main>
-      </S.LayoutWrapper>
-    );
-  }
-}
+export const Layout = (props: Props) => {
```

Comment:
```suggestion
export const Layout = ({ children, componentName, location }: Props) => {
```
Q:
How to decide if you should write `const { children, componentName, location } = props; `?
A:
Destructuring of props in component's arguments make `export const Layout = (props: Props) => {` break to another line?
No? -> `export const Layout = ({ children, componentName, location }: Props) => {`
Yes? -> `const { children, componentName, location } = props;`
Have too much props (like `const { children, componentName, location, componentName2, location2, componentName3, location3, componentName4, location 4, location3, componentName4, location 4} = props;` ) ? -> destructure in 2 lines
```
const { children, componentName, location, componentName2, location2, } = props;
const { componentName3, location3, componentName4, location 4 componentName4, location 4} = props;
```